sdcm/utils/raft/common.py

```diff
@@ -0,0 +1,39 @@
+import logging
+
+
+LOGGER = logging.getLogger(__name__)
+
+
+def check_raft_ready_on_nodes(nodes: list["BaseNode"]) -> bool:
+    LOGGER.debug("Check raft feature state on node")
+
+    nodes_raft_status = []
+    for node in nodes:
+        if raft_ready := node.raft.is_ready():
+            nodes_raft_status.append(raft_ready)
+            continue
+        LOGGER.error("Node %s has raft state: %s", node.name, node.raft.get_status())
+
+    if not all(nodes_raft_status):
+        raise Exception("Raft is not ready")
+    LOGGER.debug("Raft is ready!")
+    return True
+
+
+def check_raft_enabled_on_nodes(nodes: list["BaseNode"]) -> bool:
+    LOGGER.debug("Check raft is enabled on nodes")
+    raft_enabled_state = [node.raft.enabled for node in nodes]
+    LOGGER.debug("Raft feature state: %s", raft_enabled_state)
+    raft_enabled = all(raft_enabled_state)
+    raft_disabled = all(map(lambda x: not x, raft_enabled_state))
+    if not raft_enabled and not raft_disabled:
+        raise Exception("Raft is not enabled on all nodes")
+
+    LOGGER.debug("Raft feature: %s on all nodes", raft_enabled)
+    return raft_enabled
+
+
+def validate_raft_on_nodes(nodes: list["BaseNode"]):
+    LOGGER.debug("Check raft on cluster")
```

Comment:
This log message is redundant. Below 2 functions print their own similar messages.

Reply:
removed it